packages/scripts/utils/config.js

```diff
@@ -17,7 +17,21 @@ const hasJestConfig = () =>
 	hasProjectFile( 'jest.config.json' ) ||
 	hasPackageProp( 'jest' );
 
+const hasWebpackConfig = () => hasCliArg( '--config' ) ||
+	hasProjectFile( 'webpack.config.js' ) ||
+	hasProjectFile( 'webpack.config.babel.js' );
+
+const getWebpackArgs = ( additionalArgs = [] ) => {
+	const webpackArgs = [ ...getCliArgs() ];
+	if ( ! hasWebpackConfig() ) {
+		webpackArgs.push( ...[ '--config', fromConfigRoot( 'webpack.config.js' ) ] );
```

Comment:
Nit: [array.push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push#Syntax) can take muliple arguments so there is no need to use array spread here.

```suggestion
		webpackArgs.push( '--config', fromConfigRoot( 'webpack.config.js' ) );
```